README.md

```diff
@@ -1,2 +1,44 @@
 # prod-wgs-germline-snps-indels
-Workflows used in production at Broad for germline short variant discovery in WGS data
+### Purpose : 
+Workflows used in production at Broad for germline short variant discovery in WGS data 
+
+### PairedSingleSampleWF :
+This WDL pipeline implements data pre-processing and initial variant calling (GVCF
+generation) according to the GATK Best Practices (June 2016) for germline SNP and
+Indel discovery in human whole-genome sequencing (WGS) data.
+
+#### Requirements/expectations
+- Human whole-genome pair-end sequencing data in unmapped BAM (uBAM) format
+- One or more read groups, one per uBAM file, all belonging to a single sample (SM)
+- Input uBAM files must additionally comply with the following requirements:
+- - filenames all have the same suffix (we use ".unmapped.bam")
+- - files must pass validation by ValidateSamFile
+- - reads are provided in query-sorted order
+- - all reads must have an RG tag
+- Reference genome must be Hg38 with ALT contigs
+#### Outputs 
```

Comment:
It would be super-helpful if this and the PairedEndSingleSampleWf.md were said more about the outputs.
For example, this top-level README.md could indicate *something* like:

#### Outputs
- GVCF
- CRAM
- Metrics/QC reports

and then the PairedEndSingleSampleWf.md could be more detailed about the CRAM (binned quality scores, but otherwise lossless) and some details about the QC metrics and reports.

Reply:
By details on CRAM and QC metrics, do you mean listing all the files being generated?

Reply:
Thanks. I had meant a little more descriptive, but what you have added is very helpful.

For example, instead of:

    unsorted_read_group_base_distribution_by_cycle_pdf
    unsorted_read_group_base_distribution_by_cycle_metrics 
    unsorted_read_group_insert_size_histogram_pdf 
    unsorted_read_group_insert_size_metrics 
    unsorted_read_group_quality_by_cycle_pdf 
    unsorted_read_group_quality_by_cycle_metrics 
    unsorted_read_group_quality_distribution_pdf
    unsorted_read_group_quality_distribution_metrics 

I was thinking:

    Unsorted read group metrics and PDFs for:

    - Base distribution by cycle
    - Insert size histogram
    - Quality by cycle
    - Quality distribution

Reply:
Done

Reply:
Thanks!